.gitignore

```diff
@@ -5,3 +5,9 @@
 # https://digitalfortress.tech/tricks/creating-a-global-gitignore/
 
 build
+
+# tools
```

Comment:
why are we ignoring all the tools? We would like maybe to develop those.

Reply:
https://github.com/mainflux/mainflux/pull/1741#discussion_r1154588982